hack/Python-sdk/README.md

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
can we have this by /hack/python-sdk?

Reply:
@ellis-bigelow Thanks, updated to `python-sdk`.